lms/static/scripts/frontend_apps/components/FilePickerFormFields.js

```diff
@@ -13,6 +13,7 @@ import { contentItemForContent } from '../utils/content-item';
  * @prop {Record<string,string>} formFields - Form fields provided by the backend
  *   that should be included in the response without any changes
  * @prop {string|null} groupSet
+ * @prop {string|null} extLTIAssignmentId
```

Comment:
Is there a good piece documentation somewhere in the backend code about what this parameter means?